290. Word Pattern.py

```diff
@@ -0,0 +1,34 @@
+from collections import OrderedDict
+
+class Solution:
+    def wordPattern(self, pattern, str):
+        """
+        :type pattern: str
+        :type str: str
+        :rtype: bool
+        """
+        letters = [] #letters from pattern
+        words = [] #unique words
+        sentence = str.split() #each word
+        od = OrderedDict()
+        
+        for word in sentence:
+            if word not in words:
+                words.append(word)
+        
+        for letter in pattern:
```

Comment:
Same thing as above. When we pass a single string to a set it will create a set of the letters.
`letters = set(pattern)`